adafruit_led_animation/sequence.py

```diff
@@ -275,3 +274,49 @@ def show(self):
         Draws the current animation group members.
         """
         self.current_animation.show()
+
+
+class OneShot(AnimationSequence):
```

Comment:
I like it! Thanks for adding this. However, I think the name could be clearer. I get where you're going with it but I think something more descriptive and in line with the other class names in the library would make more sense. I'm struggling to come up with a suggestion though.

Reply:
`AnimateOnce` perhaps.